docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
If we want to call it from node_install we would need to rewrite the bash script to install yarn in zulip-npm-cache and all the calls to call yarn from there. I could do that if you think that would be better long term.